events/elections/2021/election.yaml

```
@@ -0,0 +1,17 @@
+name: 2021 Steering Committee Election
+organization: Kubernetes
+start_datetime: 2021-10-07 00:00:01
+end_datetime: 2021-10-28 23:59:00
+no_winners: 4
+allow_no_opinion: True
+delete_after: True
+show_candidate_fields:
+  - employer
+  - slack
+election_officers:
+  - jberkus
+  - alisondy
+  - coderanger
+eligibility: All Kubernetes contributors with 50 or more contributions (per Devstats) in the prior year may vote.
```

```suggestion
eligibility: Members of one or more of the Kubernetes GitHub Orgs who has at least 50 contributions to the Kubernetes project over the past year, according to a snapshot taken YYYY-MM-DD of the data driving the devstats developer activity counts dashboard.
```

Just wanted to be explicit with the Kubernetes Org requirement here as well 👍 
I'd also suggest an explicit date for the devstats data here and elsewhere since prior year could be interpreted different ways e.g. 2020-01-01 to 2020-12-31, when (for last year) it was 2019-08-01 to 2020-08-01.

---

Given the UI that description text isn't meant to be more that one line long; not sure how wonky it'll look if it's a full paragraph.  It's also meant to be a shorthand version of the eligibility. More inclined to put "Kubernetes Org members with more than 50 contributions in the prior year are eligible to vote.  See the [election guide]()"

---

Updated, see new wording.